### PR TITLE
Add kafka-messaging module.

### PR DESCRIPTION
**Pull request by Alexander-Miller**

### CODE REVIEW

File: spring-boot/messaging-kafka/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+    implementation("org.jetbrains.kotlin:kotlin-reflect")
+
+    testImplementation("org.springframework.boot:spring-boot-starter-test")
+    testImplementation("org.springframework.kafka:spring-kafka-test")
+    testImplementation("io.mockk:mockk")
+    testImplementation("com.ninja-squad:springmockk")
+    testImplementation("org.testcontainers:testcontainers")
+    testImplementation("org.testcontainers:kafka:1.16.2")
```
Comment (slu-it) [inline spring-boot/messaging-kafka/build.gradle.kts:23]: version should be managed in root build file

File: spring-boot/messaging-kafka/src/main/kotlin/kafka/books/Examples.kt [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+object Examples {
+
+    val cleanCode = BookRecord(
+            id = UUID.fromString("b3fc0be8-463e-4875-9629-67921a1e00f4"),
+            book = Book(
+                    isbn = "9780132350884",
+                    title = "Clean Code"
+            )
+    )
```
Comment (slu-it) [inline spring-boot/messaging-kafka/src/main/kotlin/kafka/books/Examples.kt:13]: doubled indentation?

**Review by slu-it — CHANGES_REQUESTED** [review]

**Comment by slu-it** [comment]
Also, pleas add a CI build job in `./.github/workflows/ci.yml`

File: spring-boot/messaging-kafka/build.gradle.kts (hunk 1 above)
Comment (Alexander-Miller) [reply spring-boot/messaging-kafka/build.gradle.kts:23]: done

File: spring-boot/messaging-kafka/src/main/kotlin/kafka/books/Examples.kt (hunk 2 above)
Comment (Alexander-Miller) [reply spring-boot/messaging-kafka/src/main/kotlin/kafka/books/Examples.kt:13]: done

**Comment by Alexander-Miller** [comment]
done

**Review by slu-it — APPROVED** [review]

**Pull request merged**